bin/dev/install_tools.sh

```diff
@@ -1,90 +1,49 @@
 echo "Fetching fissile $fissile_url ..."
 wget -q "$fissile_url"   -O - | tar xz --to-stdout fissile > "${FISSILE_BINARY}"
-echo "Installed: $("${FISSILE_BINARY}" version)"
 
 echo "Fetching stampy $stampy_url ..."
-wget -q "$stampy_url"   -O - | tar xz -C "${bin_dir}" stampy
-
-echo "Unpacking cf CLI ..."
-tar -xzf "${tools_dir}/cf.tgz" -C "${bin_dir}"
-
-echo "Fetching kubectl ${kubectl_url} ..."
-wget -q "${kubectl_url}" -O "${bin_dir}/kubectl"
-wget -q "${k_url}" -O "${bin_dir}/k"
-wget -q "${kk_url}" -O "${bin_dir}/kk"
-
-echo "Fetching helm from ${helm_url} ..."
-wget -q "${helm_url}" -O - | tar xz -C "${bin_dir}" --strip-components=1 linux-amd64/helm
+wget -q "$stampy_url"   -O - | tar xz -C "${SCF_BIN_DIR}" stampy
 
 echo "Making binaries executable ..."
 chmod a+x "${FISSILE_BINARY}"
-chmod a+x "${bin_dir}/stampy"
-chmod a+x "${bin_dir}/cf"
-chmod a+x "${bin_dir}/kubectl"
-chmod a+x "${bin_dir}/k"
-chmod a+x "${bin_dir}/kk"
-chmod a+x "${bin_dir}/helm"
+chmod a+x "${SCF_BIN_DIR}/stampy"
 
-echo "Installing certstrap ..."
-# We run chown in docker to avoid requiring sudo
-docker run --rm -v "${bin_dir}":/out:rw "golang:${GOLANG_VERSION}" /usr/bin/env GOBIN=/out go get github.com/square/certstrap
-if [[ $(stat -c '%u' "${bin_dir}/certstrap") -eq 0 ]]; then
-  docker run --rm -v "${bin_dir}":/out:rw "golang:${GOLANG_VERSION}" /bin/chown "$(id -u):$(id -g)" /out/certstrap
-fi
+echo "Installed: $("${FISSILE_BINARY}" version)"
 
 echo "Pulling ruby bosh image ..."
 docker pull splatform/bosh-cli
 
-# Note that we might not have a k8s available; do this only if we're in vagrant
-if systemctl is-active kube-apiserver.service ; then
-  echo "Installing tiller for helm ..."
-  helm init
-else
-  echo "Skipping tiller installation for helm; no local kube found"
-fi
-
 echo "Installing helm-certgen ..."
 helm_certgen_dir="$(mktemp -d)"
 trap "rm -rf '${helm_certgen_dir}'" EXIT
 git clone --branch "${HELM_CERTGEN_VERSION}" --depth 1 https://github.com/SUSE/helm-certgen.git "${helm_certgen_dir}"
-docker run --rm -v "${bin_dir}":/out:rw -v "${helm_certgen_dir}:/go/src/github.com/SUSE/helm-certgen:ro" "golang:${GOLANG_VERSION}" /usr/bin/env GOBIN=/out go get github.com/SUSE/helm-certgen
-if [[ $(stat -c '%u' "${bin_dir}/helm-certgen") -eq 0 ]]; then
-  docker run --rm -v "${bin_dir}":/out:rw "golang:${GOLANG_VERSION}" /bin/chown "$(id -u):$(id -g)" /out/helm-certgen
+docker run --rm -v "${SCF_BIN_DIR}":/out:rw -v "${helm_certgen_dir}:/go/src/github.com/SUSE/helm-certgen:ro" "golang:${GOLANG_VERSION}" /usr/bin/env GOBIN=/out go get github.com/SUSE/helm-certgen
+if [[ $(stat -c '%u' "${SCF_BIN_DIR}/helm-certgen") -eq 0 ]]; then
+  docker run --rm -v "${SCF_BIN_DIR}":/out:rw "golang:${GOLANG_VERSION}" /bin/chown "$(id -u):$(id -g)" /out/helm-certgen
 fi
 mkdir -p "${HOME}/.helm/plugins" # Necessary if we didn't run `helm init`
 rm -rf "${HOME}/.helm/plugins/certgen"
 mv --no-target-directory "${helm_certgen_dir}/plugin" "${HOME}/.helm/plugins/certgen"
```

Comment:
Right; sorry for not being clear, but I was thinking that the helm-touching bits should be run as a not-root user (in vagrant) so that we don't have to think so hard about permissions issues. (For not-vagrant, we will still end up whatever user it happens to be, which should be fine.)